lib/internal/test_runner/mock.js

```diff
@@ -157,16 +163,30 @@ class MockTracker {
         'options.setter', setter, "cannot be used with 'options.getter'"
       );
     }
+    const getOriginalObject = (descriptor) => {
```

Comment:
```suggestion
    function getOriginalObject(descriptor) {
```

Reply:
what's the reason for changing from a const to a function?

Reply:
Mainly for 2 reasons:

- This function did not call/access the parent scope (access through `this`). 
- Arrow functions are anonymous/unnamed functions (which do not have their function name on the error stack)